tests/resources/VCH-Util.robot

```diff
@@ -382,6 +395,8 @@ Secret Curl Container Logs
     Log  ${out}
     ${out}=  Run  curl -k -b vic-admin-cookies %{VIC-ADMIN}/container-logs.zip -o ${SUITE NAME}-%{VCH-NAME}-container-logs${name-suffix}.zip
     Log  ${out}
+    ${out}=  Run  curl -k -b vic-admin-cookies %{VIC-ADMIN}/logs/port-layer.log
+    Should Not Contain  ${out}  SIGSEGV: segmentation violation
```

Comment:
🤔

Reply:
yeah, what? Does the port layer log sometimes break `curl`??

Reply:
I think that's a check for `panic` - checking for the Go panic decorations is probably more portable across error types. Should have caught that when reviewing.